types/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "mc-bomb-types"
+name = "grapevine-types"
```

Comment:
:question: Should the crate use the `mc-` prefix? Thinking of https://crates.io/crates/grapevine
If so the readme links should be updated too
```suggestion
name = "mc-grapevine-types"
```

Reply:
i guess it should, will fix